### PR TITLE
ipn/ipnlocal: add tags and a few other details to self status

### LINKED ISSUE #6076: FR: Machine's own tags should be available in Self field when running `tailscale up --json`
### What are you trying to do?

I'm trying to install some automation script on my servers that discovers what its capabilities are on my tailnet based on what ACL tags are assigned to the node it's running on. It also should not have to directly contact tailscale control api for this information, as that would require distributing API keys which is less than ideal.

### How should we solve this?

Populate Self.Tags when querying local-api for Status

### What is the impact of not solving this?

I have to manually signal my script with capabilities information on a machine after it connects to the tailnet, which gets tedious when onboarding new devices to my network.

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by knyar**
This makes tags, creation time, exit node option and primary routes for the current node exposed via `tailscale status --json`

Fixes #6076

Signed-off-by: Anton Tolchanov <anton@tailscale.com>

### CODE REVIEW

File: ipn/ipnlocal/local.go [hunk 1]
```diff
@@ -592,6 +593,14 @@ func (b *LocalBackend) updateStatus(sb *ipnstate.StatusBuilder, extraLocked func
 				if c := sn.Capabilities; len(c) > 0 {
 					ss.Capabilities = append([]string(nil), c...)
 				}
+				if sn.Tags != nil {
+					v := views.SliceOf(sn.Tags)
+					ss.Tags = &v
+				}
+				ss.Created = sn.Created
+				ss.ExitNodeOption = tsaddr.PrefixesContainsFunc(sn.AllowedIPs, func(r netip.Prefix) bool {
```
Comment (maisem) [inline ipn/ipnlocal/local.go:601]: tsaddr.ContainsExitRoutes

File: ipn/ipnlocal/local.go [hunk 2]
```diff
@@ -592,6 +593,14 @@ func (b *LocalBackend) updateStatus(sb *ipnstate.StatusBuilder, extraLocked func
 				if c := sn.Capabilities; len(c) > 0 {
 					ss.Capabilities = append([]string(nil), c...)
 				}
+				if sn.Tags != nil {
```
Comment (maisem) [inline ipn/ipnlocal/local.go:596]: We should also add PrimaryRoutes

Comment (maisem) [reply ipn/ipnlocal/local.go:596]: or maybe we should consolidate this with `populatePeerStatusLocked` to not have a lot of duplicated code.

File: ipn/ipnlocal/local.go (hunk 1 above)
Comment (knyar) [reply ipn/ipnlocal/local.go]: Oh nice, I have not noticed that.

File: ipn/ipnlocal/local.go (hunk 2 above)
Comment (knyar) [reply ipn/ipnlocal/local.go]: Done

**Review by maisem — APPROVED** [review]

**Pull request merged**